src/test/scala/com/bwsw/cloudstack/entities/events/CloudStackEventTestSuite.scala

```diff
@@ -0,0 +1,94 @@
+/*
+* Licensed to the Apache Software Foundation (ASF) under one
+* or more contributor license agreements. See the NOTICE file
+* distributed with this work for additional information
+* regarding copyright ownership. The ASF licenses this file
+* to you under the Apache License, Version 2.0 (the
+* "License"); you may not use this file except in compliance
+* with the License. You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an
+* "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+* KIND, either express or implied. See the License for the
+* specific language governing permissions and limitations
+* under the License.
+*/
+package com.bwsw.cloudstack.entities.events
+
+import java.util.UUID
+
+import com.bwsw.cloudstack.entities.common.JsonMapper
+import com.bwsw.cloudstack.entities.events.Constants._
+import com.bwsw.cloudstack.entities.events.account.{AccountCreateEvent, AccountDeleteEvent}
+import com.bwsw.cloudstack.entities.events.user.UserCreateEvent
+import com.bwsw.cloudstack.entities.events.vm.{VirtualMachineCreateEvent, VirtualMachineDestroyEvent}
+import org.scalatest.FlatSpec
+
+class CloudStackEventTestSuite extends FlatSpec {
+  val status = Statuses.COMPLETE
+
+  it should "be deserialized to the UserCreateEvent if event is USER.CREATE" in {
```

Comment:
use Constants.Events instead of "USER.CREATE" string

Reply:
ok